build/tailscale/files/tailscale.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
```

Comment:
please run `tools/tidy-manifests` on the mainfest to format it consistently with other manifests